Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.1"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Recommendation = "d27d45ae-66f1-5d53-80d2-11744f5c9556"
```

Comment:
Is this a dependency for a new feature?
If so, perhaps put it on a different branch.